apps/dev-recruiters/src/pages/[slug]/index.tsx

```diff
@@ -5,11 +5,13 @@ import { GetStaticPaths, GetStaticProps } from 'next';
 import Head from 'next/head';
 import { ThemeProvider } from 'styled-components';
 import ProjectDetails from '../../components/modules/DetailedPage';
+import { agent } from '@devlaunchers/utility';
 
 export const getProjectsSlugs = async () => {
-  const res = await fetch(
-    `${process.env.NEXT_PUBLIC_STRAPI_URL}/projects?_publicationState=live`
-  );
+  // const res = await fetch(
+  //   `${process.env.NEXT_PUBLIC_STRAPI_URL}/projects?_publicationState=live`
+  // );
+  // const res = await agent.Projects.list();
```

Comment:
any reason we are NOT using the agent here?

Reply:
old implementation, i removed that block.